numpy/core/src/multiarray/multiarraymodule.c

```diff
@@ -4681,6 +4682,10 @@ NPY_VISIBILITY_HIDDEN PyObject * npy_ma_str_numpy = NULL;
 static int
 intern_strings(void)
 {
+    npy_ma_str_current_allocator = PyUnicode_InternFromString("npy_ma_str_current_allocator");
```

Comment:
```suggestion
    npy_ma_str_current_allocator = PyUnicode_InternFromString("current_allocator");
```
Unless there is a reason to prefix it?  But then I would probably use `numpy._current_allocator` or so.